x-pack/filebeat/module/crowdstrike/falcon/test/falcon-events.log-expected.json

```diff
@@ -0,0 +1,572 @@
+        "input.type": "log",
+        "crowdstrike.metadata.customerIDString": "8f69fe9e-b995-4204-95ad-44f9bcf75b6b",
+        "crowdstrike.metadata.offset": 294564,
+        "crowdstrike.metadata.eventType": "DetectionSummaryEvent",
+        "crowdstrike.metadata.eventCreationTime": 1582101000000,
+        "crowdstrike.metadata.version": "1.0",
+        "crowdstrike.event.ParentProcessId": 38682494050,
+        "crowdstrike.event.SHA256String": "6a671b92a69755de6fd063fcbe4ba926d83b49f78c42dbaeed8cdb6bbc57576a",
+        "crowdstrike.event.SensorId": "7c808b4c8878433287eea53d4a8c3268",
+        "crowdstrike.event.LocalIP": "192.168.12.51",
+        "crowdstrike.event.FalconHostLink": "https://falcon.crowdstrike.com/ec86abd353824e96765ecbe18eb4f0b4",
+        "crowdstrike.event.Tactic": "Malware",
+        "crowdstrike.event.ProcessEndTime": 0,
+        "crowdstrike.event.Severity": 4,
+        "crowdstrike.event.CommandLine": "C:\\Windows\\Explorer.EXE",
+        "crowdstrike.event.Technique": "Ransomware",
+        "crowdstrike.event.Objective": "Falcon Detection Method",
+        "crowdstrike.event.ProcessId": 38684386611,
+        "crowdstrike.event.DetectDescription": "Terminated a process related to the deletion of backups, which is often indicative of ransomware activity.",
+        "crowdstrike.event.PatternDispositionDescription": "Prevention, process killed.",
+        "crowdstrike.event.PatternDispositionFlags.Indicator": false,
+        "crowdstrike.event.PatternDispositionFlags.Detect": false,
+        "crowdstrike.event.PatternDispositionFlags.KillProcess": true,
+        "crowdstrike.event.PatternDispositionFlags.KillSubProcess": false,
+        "crowdstrike.event.PatternDispositionFlags.KillParent": false,
+        "crowdstrike.event.PatternDispositionFlags.OperationBlocked": false,
+        "crowdstrike.event.PatternDispositionFlags.ProcessBlocked": false,
+        "crowdstrike.event.PatternDispositionFlags.InddetMask": false,
+        "crowdstrike.event.PatternDispositionFlags.SensorOnly": false,
+        "crowdstrike.event.PatternDispositionFlags.Rooting": false,
+        "crowdstrike.event.PatternDispositionFlags.QuarantineMachine": false,
+        "crowdstrike.event.PatternDispositionFlags.QuarantineFile": false,
+        "crowdstrike.event.PatternDispositionFlags.PolicyDisabled": false,
+        "crowdstrike.event.FileName": "explorer.exe",
+        "crowdstrike.event.MachineDomain": "CORP-DOMAIN",
+        "crowdstrike.event.PatternDispositionValue": 16,
+        "crowdstrike.event.ComputerName": "alice-laptop",
+        "crowdstrike.event.UserName": "alice",
+        "crowdstrike.event.MD5String": "ac4c51eb24aa95b77f705ab159189e24",
+        "crowdstrike.event.DetectId": "ldt:ec86abd353824e96765ecbe18eb4f0b4:38655257584",
+        "crowdstrike.event.MACAddress": "xx-xx-xx-xx-xx",
+        "crowdstrike.event.ProcessStartTime": 1536846339,
+        "crowdstrike.event.DetectName": "Process Terminated",
```

Comment:
Would it make sense to map this one to `rule.name`? This one doesn't sound like some intricate rule, and we don't have other samples filling this one. But based on the definition of the field:

> Name of the detection.

Same for `DetectDescription` => `rule.description`

Reply:
I'm good with this. Change made.